src/server.tsx

```diff
@@ -57,13 +59,15 @@ interface Epic {
     css: string;
 }
 
+const [, fetchDefaultEpicContentCached] = memoise(fetchDefaultEpicContent);
```

Comment:
I wasn't aware of this syntax before, but it's the way to ignore positional parameters when destructuring an array.

Reply:
Ah cool, me neither. I think in the past I've done something like capture into a variable with a leading underscore to signify that I'm not using it. This is definitely better!

Reply:
Yeah, the underscore still counts as a legit variable name unfortunately, so you'll get a lint warning about an unused variable.